.changeset/violet-chicken-smell.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-azure-sites-backend': minor
+---
+
+Azure Sites list now hides the internal/microsoft only `hidden-` tags from the list of tags that are returned. Updated the logstreamHref to /logstream-quickstart rather than just /logstream to stream logs in the current Azure Portal UI.
```

Comment:
```suggestion
Azure Sites list now hides the internal/microsoft only `hidden-` tags from the list of tags that are returned. Updated the log endpoint to /logstream-quickstart rather than just /logstream to stream logs in the Azure Portal UI.
```